sources/web/datalab/polymer/test/datalab-toolbar-test.ts

```diff
@@ -21,13 +21,47 @@
 describe('<table-preview>', () => {
   let testFixture: ToolbarElement;
 
-  beforeEach(() => {
+  before(() => {
+    SettingsManager.getAppSettingsAsync = () => {
+      const mockSettings: common.AppSettings = {
+        allowHttpOverWebsocket: true, allowOriginOverrides: [], configUrl:
```

Comment:
This is pretty ugly. I suppose we can fix later if you want.

Reply:
Missed this.
I think it's fine for settings, especially if most of them are just there for the build.
The fix is to make all settings definitions in common.d.ts optional, so we can just specify the ones we want here.